sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -81,19 +81,23 @@
 * Fixed NPE on `getDiagnostics` for `CosmosStoredProcedureResponse`.
 * Fixed empty `resourceAddress` in `CosmosException`.
 
-### 4.8.0 (2020-10-27)
+## 4.8.0 (2020-10-27)
 #### New Features
 * Added `contentResponseOnWriteEnabled` feature to `CosmosItemRequestOptions`.
 
 #### Key Bug Fixes
 * Fixed an issue which may affect query behaviour when resuming from a continuation token.
 
-### 4.7.1 (2020-10-21)
+## 4.7.1 (2020-10-21)
 #### Key Bug Fixes
 * Improved the 449 retry policy to force back-off on initial retry and start with shorter back-offs.
 
+<<<<<<< HEAD
```

Comment:
@chidozieononiwu FYI this looks like a overlooked bad merge. 

I just chatted with @kushagraThapar and he is planning to fix this and also revert the heading changes to the cosmos changelogs as these need to be level 3 as they are directly pointed to docs site https://docs.microsoft.com/en-us/azure/cosmos-db/sql-api-sdk-java-v4. We need to figure out how we that docs requirement with our changelog guidelines.